web/viewer.html

```diff
@@ -27,6 +27,8 @@
 <!--#if GENERIC || CHROME-->
     <meta name="google" content="notranslate">
 <!--#endif-->
+    <!-- Ignore legacy IE compatibility settings and prefer HTML5 -->
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
Please also address the review comments in https://github.com/mozilla/pdf.js/pull/6374#issuecomment-136528408, i.e. change the patch to:
```html
<!--#if GENERIC-->
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
<!--#endif-->
```
After this, please squash the commits, see https://github.com/mozilla/pdf.js/wiki/Squashing-Commits.